### PR TITLE
Add Perl Implementation

### PR DESCRIPTION
**Pull request by robaganrab**
An implementation in Perl.

kumar-asshole.pl depends on [Mail::Webmail::Gmail](https://metacpan.org/pod/Mail::Webmail::Gmail), which is outdated currently. Other scripts should work with correct config.

Implements #51

### CODE REVIEW

**Comment by NARKOZ** [comment]
Thank you!

File: perl/fucking-coffee.pl [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
```
Comment (jcutbank) [inline perl/fucking-coffee.pl]: Ho

Comment (jcutbank) [inline perl/fucking-coffee.pl]: Hey

Comment (jcutbank) [inline perl/fucking-coffee.pl]: Fuck off

**Pull request merged**